src/basho_bench_worker.erl

```diff
@@ -102,12 +106,17 @@ init([SupChild, Id]) ->
     Ops     = ops_tuple(),
     ShutdownOnError = basho_bench_config:get(shutdown_on_error, false),
 
+    %% Check configuration for flag enabling new API that passes opaque State object to support accessor functions
+    ApiPassState = basho_bench_config:get(api_pass_state),
```

Comment:
If you notice the one line change to basho_bench.app.src it sets the default to false automatically so it isn't undefined. 
Meant to be a transitional flag so we can make supporting code changes after this and not mess up anything CDL is doing.  

What I found testing it was I could add new/2 and run/3 wrappers to the existing source so it could be called either way.